src/Model/NewsCategoryModel.php

```diff
@@ -137,21 +137,25 @@ public static function findPublishedByArchives(array $archives, array $ids = [])
      */
     public static function findPublishedByIdOrAlias($idOrAlias)
     {
-        $values = [$idOrAlias];
+        $values = [];
+        $columns = [];
         $t = static::getTableAlias();
 
         // Determine the alias condition
-        if (MultilingualHelper::isActive()) {
-            $aliasCondition = 't1.alias=? OR t2.alias=?';
-            $values[] = $idOrAlias;
+        if (is_numeric($idOrAlias)) {
+            $columns[] = "$t.id=?";
             $values[] = $idOrAlias;
         } else {
-            $aliasCondition = "$t.alias=?";
-            $values[] = $idOrAlias;
+            if (MultilingualHelper::isActive()) {
+                $columns[] = 't1.alias=? OR t2.alias=?';
```

Comment:
Can you please wrap it in the brackets?

Reply:
Do you mean `$columns = ['…'];` instead of `$columns[] = '…'`? Sure :)

Reply:
No, I meant this `'(t1.alias=? OR t2.alias=?)'` :wink: